.github/workflows/builds.yml

```
@@ -32,25 +32,25 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, macos-10.15]
-        target: [debug, optim]
-        mpi: [sequential]
+        target: [dbg, opt]
+        mpi: [seq]
         build-system: [cmake]
         include:
           - os: ubuntu-20.04
-            target: optim
-            mpi: sequential
+            target: opt
+            mpi: seq
             build-system: make
           - os: ubuntu-20.04
-            target: debug
-            mpi: parallel
+            target: dbg
+            mpi: par
             build-system: make
           - os: macos-10.15
-            target: optim
-            mpi: sequential
+            target: opt
+            mpi: seq
             build-system: make
           - os: macos-10.15
-            target: debug
-            mpi: parallel
+            target: dbg
+            mpi: par
             build-system: make
     name: ${{ matrix.os }}-${{ matrix.target }}-${{ matrix.mpi }}-${{ matrix.build-system }}
```

For consistency with MFEM:

```suggestion
    name: ${{ matrix.os }}-${{ matrix.build-system }}-${{ matrix.target }}-${{ matrix.mpi }}
```